templates/GenespotRE/dashboard.html

```diff
@@ -37,12 +37,12 @@ <h1 class="page-header">Your Dashboard</h1>
                     </li>
                     <li class="item">
                         <!--TODO: get bar chart-->
-                        <img class="img-fluid" src="{% static 'img/scatter-plot.png' %}">
+                        <img class="img-fluid" src="{% static 'img/histogram.png' %}">
                         <p>Scatter Plot</p>
                     </li>
                     <li class="item">
                         <!--TODO: get bar chart-->
-                        <img class="img-fluid" src="{% static 'img/violin-plot.png' %}">
+                        <img class="img-fluid" src="{% static 'img/histogram.png' %}">
```

Comment:
Yes. The violin-plot and scatter-plot images I gathered are not good. I think they should change them anyway.